proto/minder/v1/minder.proto

```diff
@@ -990,6 +1004,20 @@ message VerifyProviderTokenFromResponse {
     string status = 1;
 }
 
+// VerifyProviderCredentialRequest contains the enrollment nonce (aka state) that was used when enrolling the provider
+message VerifyProviderCredentialRequest {
+    Context context = 1;
+
+    // enrollment_nonce is the state parameter returned when enrolling the provider
+    string enrollment_nonce = 2;
+}
+
+// VerifyProviderCredentialRequest responds with an OK status if the credential was found, KO if the credential was not
+// found or an error if an error occurred during lookup
```

Comment:
The string values for `VerifyProviderTokenFrom` are a bit funky -- it feels like we might want to change the `Status` to be something like `ProviderName`, with the following states:

* `""` if the supplied nonce is not active
* the string name of the provider if the nonce has been activated.

Reply:
That's true, I updated this to return a `created` boolean and the `providerName`. Eventually we can use the provider name to print "Provide XYZ successfully enrolled" from the CLI, but I haven't made that change in this PR.